bin/utils/check_utils.sh

```diff
@@ -34,3 +33,46 @@ function check_file_status() {
 
 
 
+# This function exits 0 when amount of ES entries is bigger than given value in a specified index.
+#arg1: Grafana access token
+#arg2: ES index (e.g. monit_es_condor)
+#arg3: ES index time extension (e.g. RecordTime/timestamp)
+#arg4: minimum number of documents to compare
+
+function check_cmp_es_hits {
+    # get monit if it doesn't exist
+    # path is subject to change
+    if [ ! -f /tmp/monit ]; then
+        curl -LJO https://github.com/dmwm/CMSMonitoring/releases/latest/download/cmsmon-tools.tar.gz
+        tar -xvzf cmsmon-tools.tar.gz
+        cp cmsmon-tools/monit /tmp/
+        rm -rf cmsmon-tools*
+    fi
+
+    TIME_EXT=$3
+    BASE_QUERY=$(echo '{"query":{"range":{"data.$TIME_EXT":{"gte":"now-1h"}}}}' | sed -e "s/\$TIME_EXT/$TIME_EXT/g")
+    JQ_RESULT=$(exec /tmp/monit -token $1 -query=$BASE_QUERY -dbname=$2 -esapi=_count | jq -re '.count')
+    if [ $JQ_RESULT -lt $4 ]; then
+        exit 1
+    fi
+}
+
+
+# This function exits 0 when specified file was modified during last N seconds (Unix time).
+#arg1: hdfs directory and file
+#arg2: time treshold in seconds
+function check_hdfs_mod_date {
+
+    CURTIME=$(date +%s)
+    # convert milliseconds to seconds
+    FILETIME=$(expr $(hadoop fs -stat "%Y" $1) / 1000 )
+    TIMEDIFF=$(expr $CURTIME - $FILETIME)
+
+    if [ $TIMEDIFF -gt $2 ]; then
+        # amount of time before last modification is bigger than specified treshold
+        # -> file wasn't modified during last $2 seconds
+        exit 1
```

Comment:
I forget to mentioned in the meetings. Let's choose a unique exit code for checks, so we can differentiate them from other processes. If I'm not wrong till 133, all exit codes are reserved and it's a good convention to set max exit code as 255. Hence, I may suggest `210` for all check function exit codes. If we see that exit code, we'll know that our check function is failed.